beginner_source/Pretraining_Vgg_from_scratch.py

```diff
@@ -0,0 +1,665 @@
+"""
+``Pretraining`` VGG from scratch 
+============================
+
+
+**Author:** `WoongJoon Choi <https://github.com/woongjoonchoi>`_
+
+In this tutorial, we will embark on an exciting journey to build and
+train a VGG network from scratch using Python and popular deep learning
+libraries such as PyTorch. We will dive into the details of the VGG
+architecture, understanding its components and the rationale behind its
+design.
+
+Our tutorial is designed for both beginners who are new to deep learning
+and seasoned practitioners looking to deepen their understanding of CNN
+architectures.
+
+Before you start 
```

Comment:
```suggestion
Before you start:
```

Reply:
Better: 
If you are running this in Google Colab, install ``albumentations``: